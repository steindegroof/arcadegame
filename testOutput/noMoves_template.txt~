Bestand: testOutput/NoMoves.txt

geen bewegingen.
