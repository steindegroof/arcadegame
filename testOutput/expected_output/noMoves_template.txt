Bestand: testOutput/noMoves.txt

geen bewegingen.
